pyproject.toml

```diff
@@ -8,7 +8,8 @@ requires = [
     # Workaround based on this commit:
     # https://github.com/harfbuzz/uharfbuzz/commit/9b607bd06fb17fcb4abe3eab5c4f342ad08309d7
     "setuptools>=64,<72.2.0; platform_python_implementation == 'PyPy'",
-    "setuptools>=64; platform_python_implementation != 'PyPy'"
+    "setuptools>=64; platform_python_implementation != 'PyPy'",    
+    "cmake>=3.12"
```

Comment:
This change is unrelated to the PR but it's a nice addition. File a separate PR with this and I'll be happy to merge.

Reply:
It's not needed for an external igraph but it's needed for the vendored one. There's no way to specify this to Python build backends, though so it's easier to just include the build dependency - it won't hurt to have it there even if the user ends up linking to an external igraph.